pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -97,22 +106,40 @@ var _ = Describe("podNIC", func() {
 			})
 		})
 	})
-	Context("when DHCP startup fails", func() {
+	When("DHCP config is correctly read", func() {
+		var (
+			podnic     *podNIC
+			domain     *api.Domain
+			dhcpConfig *cache.DHCPConfig
```

Comment:
unless my eyes are making a fool of me, you're not using this anywhere - other than in the setup stage.

If so, please make it a local variable.